test/widgets/page_checks.dart

```diff
@@ -9,3 +9,7 @@ extension WidgetRouteChecks on Subject<WidgetRoute> {
 extension AccountPageRouteMixinChecks on Subject<AccountPageRouteMixin> {
   Subject<int> get accountId => has((x) => x.accountId, 'accountId');
 }
+
+extension PageRouteChecks on Subject<PageRoute> {
```

Comment:
nit: let's keep the checks extensions for Flutter upstream classes gathered in `flutter_checks.dart`